lib/board.rb

```diff
@@ -37,28 +37,25 @@ def coord_array_setup
     end
   end
 
-  def array_increments?(coordinate_array)
-    first_num = coordinate_array[0]
-    coordinate_array[1, coordinate_array.count].each do |n|
-      return false if first_num + 1 != n
-      first_num = n
-    end
-    true
```

Comment:
Does removing this allow for non-incremented? Such as A1, A2, A4.

Reply:
Can we comment this out just in case before merging?

Reply:
good point, I just wrote a test to check it out and it seems to know that A1, A2, and A4 are invalid placements. Let's keep it commented out just in case and run some more tests to make sure!